content/en/guides/configuration-glossary/configuration-mode.md

```diff
@@ -9,11 +9,17 @@ position: 17
 - Type: `string`
   - Default: `universal`
   - Possible values:
-    - `'spa'`: No server-side rendering (only client-side navigation)
     - `'universal'`: Isomorphic application (server-side rendering + client-side navigation)
 
 > You can use this option to change default nuxt mode for your project using `nuxt.config.js`
 
+
+<base-alert type="warning">
+
+Deprecated: please use `ssr: false` instead of `mode: spa`
```

Comment:
If there is a link to ssr config, can we link to it?

Reply:
you mean we should create one as no there isn't one. What else should be added to ssr config page?
ssr:true which is default value is used for ssr and static sites which is really confusing and hard to document. suggestions?

Reply:
Only mentioning `ssr: false` should be enough then :) I tought we also have https://nuxtjs.org/guides/configuration-glossary/configuration-ssr but seems not.

Reply:
yes we don't have it. I will create it and just add ssr:false

Reply:
added

Reply:
Thanks :heart: Will use new page for create-nuxt-app link :) (https://github.com/nuxt/create-nuxt-app/pull/606)